style.css

```diff
@@ -1,250 +1,517 @@
+  color: #e51937;
+  transition: 1s;
+}
+
+.header__line {
+  position: relative;
+  top: 32px;
 }
 
 .top__line {
-display: flex;
-align-items: center;
-margin-left: -61px;
-margin-top: 34px;
+  width: 60px;
+  height: 2px;
 }
 
 .menu__link_modelX {
-display: flex;
-align-items: center;
-margin-left: 65px;
+  margin-right: 22px;
+  text-decoration: none;
+  color: #000;
+  transition: 1s;
 }
 
 .menu__link_model3 {
-display: flex;
-align-items: center;
-margin-left: 26px;
+  margin-right: 40px;
+  text-decoration: none;
+  color: #000;
+  transition: 1s;
 }
 .menu__link_roadster{
-  display: flex;
-  align-items: center;
-  margin-left: 44px;
+  margin-right: 42px;
+  text-decoration: none;
+  color: #000;
+  transition: 1s;
 }
 
 .menu__link_energy {
-  display: flex;
-  align-items: center;
-  margin-left: 48px;
+  margin-right: 218px;
+  text-decoration: none;
+  color: #000;
+  transition: 1s;
 }
 
 .top__Energy {
   margin-top: 5px;
-}
-
-.header__right {
-display: flex;
-justify-content: start;
-margin-left: 150px;
-margin-top: 43px;
+  transition: 1s;
 }
 
 .menu__link_shop {
-display: flex;
-align-items: center;
-margin-left: -39px;
+  margin-right: 25px;
+  text-decoration: none;
+  color: #000;
+  transition: 1s;
 }
 
 .menu__link_entrance {
-display: flex;
-align-items: center;
-margin-left: 32px;
+  margin-right: 27px;
+  text-decoration: none;
+  color: #000;
+  transition: 1s;
 }
 
 .header__burger {
-display: flex;
-align-items: center;
-width: 24px;
-height: 17px;
-background: #E51937;
-background: url(./images/Vector.png);
-margin-left: 30px;
-border-right: 0;
-border-left: 0;
-border-bottom: 0;
+  position: absolute;
+  width: 24px;
+  height: 17px;
+  background: #E51937;
+  background: url(./images/Vector.png);
+  border-right: 0;
+  border-left: 0;
+  border-bottom: 0;
+  margin-right: 0;
 } 
 
+.top {
+  display: flex;
+  flex-direction: column;
+}
+
 .top__title {
-display: flex;
-align-items: center;
-width: 370px;
-height: 10px;
-left: 535px;
-font-weight: 600;
-font-size: 90px;
-line-height: 10px;
-color: #000000;
-margin-left: 527px;
-margin-bottom: 40px;
-margin-top: 135px;
+  text-align: center;
+  height: 10px;
+  font-weight: 600;
+  font-size: 90px;
+  line-height: 10px;
+  color: #000;
+  margin-right: 142px;
+  margin-bottom: 26px;
 }
 
 .top__subtitle {
-display: flex;
-align-items: center;
-width: 250px;
-height: 16px;
-left: 600px;
-font-weight: normal;
-font-size: 15px;
-line-height: 18px;
-color: #000000;
-margin-left: 590px;
-margin-top: 43px;
-}
-
-.top__avto {
-display: flex;
-align-items: center;
-width: 968px;
-height: 450px;
-margin-left: 190px;
-margin-top: -323px;
+  text-align: center;
+  height: 16px;
+  font-weight: normal;
+  font-size: 15px;
+  line-height: 18px;
+  color: #000;
+  margin-right: 142px;
 }
 
 .top__fon {
-display: flex;
-align-items: center;
-width: 1514px;
-height: 467px;
-background: url("./images/fon.png");
-margin-top: -131px;
-margin-left: -10px;
+ position: inherit;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/Tesla3/pullRequest?prid=6027752)